apps/bfd-pipeline/bfd-pipeline-app/src/test/java/gov/cms/bfd/pipeline/app/PipelineManagerIT.java

```diff
@@ -487,11 +487,14 @@ public void runWayTooManyJobsThenStop() throws Exception {
     }
   }
 
-  /** Reduce tick time on built-in jobs, to speed test execution. */
+  /**
+   * Reduce tick time on built-in jobs, to speed test execution. Setting this too low can expose a
+   * race condition in the volunteer job where it attempts to enqueue the same job twice.
+   */
   @BeforeClass
   public static void configureTimers() {
-    VolunteerJob.VOLUNTEER_TICK_MILLIS = 10;
-    SchedulerJob.SCHEDULER_TICK_MILLIS = 10;
+    VolunteerJob.VOLUNTEER_TICK_MILLIS = 500;
+    SchedulerJob.SCHEDULER_TICK_MILLIS = 500;
```

Comment:
Instead of this, I'd have `PipelineJobRecordStore.findPendingJobs(int)` call a new `j.isEnqueued()` method (instead of calling `j.isStarted()`, like it does currently). This should deterministically solve the problem.